src/echarts.js

```diff
@@ -1142,8 +1143,26 @@ echartsProto.resize = function (opts) {
     flushPendingActions.call(this, silent);
 
     triggerUpdatedEvent.call(this, silent);
+
+    moveTooltip.call(this);
```

Comment:
Still not sure whether it is proper to handle tooltip relocation here, maybe it's better to move these logic to `TooltipView`?

Reply:
> Still not sure whether it is proper to handle tooltip relocation here, maybe it's better to move these logic to `TooltipView`?

Let me have a try.